tools/ci_build/github/android/mobile_package.required_operators.config

```diff
@@ -9,19 +9,21 @@
 # allow float, int8, uint8. operators that manipulate shapes or indices have int32 and int64 enabled internally.
 !globally_allowed_types;float,int8_t,uint8_t
 
-# ops used by the tf2onnx tflite converter. same list for opsets 12 and 13.
+# ops used by the tf2onnx tflite converter. full list for opset 12. incremental list for later opsets.
```

Comment:
Made the updates incremental so it's easier to see how much changed and have a rough idea of binary size impact.